pilosa/orm.py

```diff
@@ -475,7 +475,7 @@ class Field:
     def __init__(self, index, name, time_quantum,
                  cache_type, cache_size, int_min, int_max, keys, mutex, bool):
         validate_field_name(name)
-        if int_max < int_min:
+        if int_min is not None and int_max is not None and int_max < int_min:
```

Comment:
I don't have strong feelings about this, but should the client pass this through, and behave the same way as go-pilosa?

Reply:
ok, updated.